clients/gui-app/src/index.css

```
@@ -1280,6 +1280,126 @@
+      linear-gradient(black, black);
+    -webkit-mask-position:
+      0px 0px,
+      0px var(--chat-timeline-scroll-fade-height),
+      0px calc(100% - var(--chat-bottom-overlay-inset, 0px)),
+      right;
+    -webkit-mask-repeat: no-repeat;
+    -webkit-mask-size:
+      100% var(--chat-timeline-scroll-fade-height),
+      100%
+        calc(
+          100% - var(--chat-timeline-scroll-fade-height) -
```

Stylelint (17.14.1)

[error] Unexpected newline after "-" (scss/operator-no-newline-after)

(scss/operator-no-newline-after)